src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
```

Comment:
According to our checklist, classes, fields, constructors, methods must have access modifiers otherwise default will be used that isn't a good practice. Please, add access modifier for the method `getRandomBall()`.